app/main/forms.py

```diff
@@ -1683,7 +1685,7 @@ class Triage(StripWhitespaceForm):
             ("yes", "Yes"),
             ("no", "No"),
         ],
-        thing="yes or no",
+        thing="‘yes’ if you want us to contact you if it is an emergency",
```

Comment:
I cannot find this one in the spreadsheet.

It looks like it’s based on a pattern we use for other errors.

If someone can talk me through the scenario and the trigger for this error, I might be able to improve the content slightly.

Reply:
Scenario:
1. you are signed in to Notify
2. it is out of hours (for example in the evening)
3. you go to raise a support ticket
4. you select ‘problem’ rather than ‘question’
5. you get an extra page where we ask whether or not it’s an emergency
6. you fill in the rest of your support message as usual

If you answer ‘yes’ to 5. then we will treat the ticket as a P1 incident, set off Pagerduty, etc

Reply:
> I cannot find this one in the spreadsheet.
> 
> It looks like it’s based on a pattern we use for other errors, but I’m not sure about having 2 ‘if’s in the error.
> 
> If someone can talk me through the scenario and the trigger for this error, I might be able to improve the content slightly.

@karlchillmaid Do you still want to update the error message?

Reply:
@BlessedDev Yes. Sorry, I haven’t had time to look at this yet. I’ll pick it up tomorrow.

Reply:
@quis so the validation is more:

> Select ‘yes’ if it is an emergency

And not so much about whether or not the user wants us to contact them?

Reply:
@karlchillmaid that’s correct

Reply:
@quis nice. easy fix then.

```suggestion
        thing="‘yes’ if it is an emergency",
```